agent/agent.go

```diff
@@ -1015,7 +1014,12 @@ func (a *Agent) upsertCheckRun(ctx context.Context, tc *taskContext) error {
 		return nil
 	}
 
-	return a.comm.UpsertCheckRun(ctx, tc.task, *checkRunOutput)
+	if err = a.comm.UpsertCheckRun(ctx, tc.task, *checkRunOutput); err != nil {
+		return err
+	}
+
+	tc.logger.Task().Infof("Successfully upserted checkRun.")
```

Comment:
nit: maybe we can write check run name or id?

Reply:
We do that in buildCheckRun